issue_processing/issue_processing.py

```diff
@@ -98,26 +129,133 @@ def reformat(issue_data):
 
     # re-process all issues
     for issue in issue_data:
+
+        # empty container for issue types
+        issue["type"] = []
+
+        # empty container for issue resolutions
+        issue["resolution"] = []
+
+        # if an issue has no eventsList an empty Lists gets created
```

Comment:
Is there a reason for the plural in `empty Lists`? If not, could you please use the singular here?
In addition, adding a comma after the if-clause (i.e., after `eventsList`) will optimize the flow of reading.

This comment applies also for all the following comments regarding empty lists below.